routemodel/data_retrieval/get_elevations_point_compression.py

```diff
@@ -0,0 +1,165 @@
+import requests
+import pandas as pd
+import sys
+import os.path
+from csv import reader
+from config import API_KEY
+
+sys.path.append(os.path.dirname(sys.path[0]))
+
+BASE_URL = 'https://dev.virtualearth.net/REST/v1/'
+ELEVATIONS_FILE = os.path.join(sys.path[0], '../../routemodel/routes/heartland_elevations.csv')
+COORDINATES_FILE = os.path.join(sys.path[0], '../../routemodel/routes/heartland_coordinates.csv')
+
+# This just generates a dict object for the coordinates to resolve some type errors
+COORDINATES_LISTX = []
+COORDINATES_LISTY = []
+COORDINATES_LIST = []
+with open(COORDINATES_FILE, 'r') as read_obj:
+    # pass the file object to reader() to get the reader object
+    csv_reader = reader(read_obj)
+    # Iterate over each row in the csv using reader object
+    for row in csv_reader:
+        COORDINATES_LISTX.append(row[0])
+        COORDINATES_LISTY.append(row[1])
+    COORDINATES_LIST.append(dict(zip(COORDINATES_LISTX, COORDINATES_LISTY)))
+
+
+class ElevationDataRetrieval:
+    def __init__(self, wps: list, ht='sealevel'):
+        """
+        Initialize MapData
+        @param wps: dictionary of major waypoints inputted by user
+        @param ht: elevations response height model option

```

Comment:
Can you add quick information on what the options are?

Reply:
Hey I apologize about this I did not add more information because I didn't want to say the wrong thing I don't know much in-depth info about this module, I would need to talk about Umar about it, it'll be dealt with in the refactor hopefully?